community/modules/scheduler/schedmd-slurm-gcp-v5-login/variables.tf

```diff
@@ -308,6 +308,19 @@ variable "instance_image" {
   }
 }
 
+variable "instance_image_override" {
+  description = <<-EOD
+  A flag that designates that the user is aware that they are requesting
```

Comment:
> I think we should discuss this variable name and description a bit offline:
> 
> 1. I like that it is lexically next to instance_image so that it appears that way in documentation. I think I might choose a different word than override. Let's think a bit.

I believe I've gotten a bit closer to your intention with for this.  I changed the variable to `instance_image_compatible`, but we can discuss.

> 2. I think we should use different language than "not supported by" and more like "and must be produced with same release of Terraform module according to docs at https://X"

Understood, I believe the new language covers this in both the errors messages, variable descriptions, and documentation.

Reply:
Resolved.